docs/tutorials/otio-file-format-specification.md

```diff
@@ -62,20 +61,40 @@ In order to make the tree structure easy to traverse, we use the name "children"
 
 ## Metadata
 
-TODO: Explain how metadata works and why we do it that way.
+Timeline, Stack, Track, Clip, MediaReferece, and most other OTIO objects all have a `metadata` property.
```

Comment:
It may be worth explicitly calling out here that the `metadata` property is an arbitrarily deep JSON dictionary like structure with no enforced schema.